src/watch/watch.go

```diff
@@ -73,6 +73,7 @@ func Watch(state *core.BuildState, labels core.BuildLabels, callback CallbackFun
 			// Kill any previous process.
 			cancel()
 			ctx, cancel = context.WithCancel(parentCtx)
+			build(ctx, state, labels, callback)
```

Comment:
Not sure about this bit; delaying the build may be useful to make sure the changes get a chance to be written to disk before starting. It sort of feels wrong to me to be building sooner to try to prevent rebuilds...

Reply:
Fair enough; reverted this change such that this only includes the `go_get` fix!